convert_hf_to_gguf.py

```diff
@@ -5990,6 +6001,99 @@ def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None) -> Iter
 
         return [] # skip other tensors
 
+@ModelBase.register("DeepseekOCRForCausalLM")
+class DeepseekOCRVisionModel(MmprojModel):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        proc_fname = self.dir_model / "processor_config.json"
+
+        if proc_fname.is_file():
+            with open(proc_fname, "r") as f:
+                self.preprocessor_config = json.load(f)
+
+
+    def set_gguf_parameters(self):
```

Comment:
```suggestion
    def set_gguf_parameters(self):
```
No longer required, already handled.